.sobelow-skips

```diff
@@ -0,0 +1,7 @@
+
+1AC9A86DF980B86B43220695B6EB5461
+9EC3617CEF145CB2F5C05F768F987557
+B2DFBE05B77FB5179E439694DB527D5F
+BDACD306F582E17B549E43A4DA075296
+C15BCE8CAF49A8448E904E375BA0C396
+E421E305F738CAFD1A717A8FD2575135
```

Comment:
Some funky whitespace in here — final line doesn't end in a newline, but there's an extra newline at the start of the file. Auto-generated?

Reply:
Yep, auto-generated by `mix sobelow --mark-skip-all`, though if I open it up in my editor and add a newline at the end / remove the blank line at the beginning it still seems to work.